package.json

```diff
@@ -3,7 +3,7 @@
   "license": "AGPL-3.0-only",
   "private": true,
   "name": "grafana",
-  "version": "8.4.0-pre",
+  "version": "8.4.1-pre",
```

Comment:
Not this version file, the `.fusebit.version.json` file.

Reply:
Done